go/fixchain/fixer_test.go

```diff
@@ -126,7 +208,7 @@ func TestUpdateCounters(t *testing.T) {
 		for _, err := range test.errors {
 			ferrs = append(ferrs, &FixError{Type: err})
 		}
-		f.updateCounters(ferrs)
+		f.updateCounters(nil, ferrs)
```

Comment:
Yes, it should have gone in the last PR (1139), but that has already been merged.  It was my mistake.